app/build.gradle

```diff
@@ -134,5 +134,7 @@ dependencies {
     androidTestImplementation "com.android.support.test.espresso:espresso-core:$rootProject.espressoVersion"
     androidTestImplementation "com.android.support.test:runner:$rootProject.runnerVersion"
     androidTestImplementation "com.android.support.test:rules:$rootProject.rulesVersion"
+
```

Comment:
@ethan627hsu Add this as last implementation statement, that is above testImplementation and androidTestImplementation .

Reply:
Ok, added the change @miPlodder